contracts/v2/AggLayerGateway.sol

```diff
@@ -37,6 +37,9 @@ contract AggLayerGateway is
     bytes32 internal constant AL_FREEZE_PP_ROUTE_ROLE =
         keccak256("AL_FREEZE_PP_ROUTE_ROLE");
 
+    // Current AggLayerGateway version
+    string public constant AGGLAYERGATEWAY_VERSION = "al-v0.3.0";
```

Comment:
```suggestion
    string public constant AGGLAYER_GATEWAY_VERSION = "al-v0.3.0";
```

Reply:
I'd suggest to follow our own SC versioning, not dependant on the `al`.
Therefore, I'd set `v1.0.0` since the SC is on production and has not been yet modified